src/olympia/lib/settings_base.py

```diff
@@ -1635,3 +1635,9 @@ def read_only_mode(env):
         'manifest_path': STATIC_BUILD_MANIFEST_PATH,
     }
 }
+
+# The environment in which the application is running.
+# This is set by the environment variables in production environments.
+# For local it is hard coded to "local" in `settings.py` to guarantee a clear
```

Comment:
I don't see a hard coded 'local' value in `settings.py`

Reply:
It is defined in docker-compose.yml. Updated.